### PR TITLE
Add a little style to commit messages

### PR DESCRIPTION
**Pull request by slang25**
I'm sweating the big stuff here

### CODE REVIEW

**Comment by slang25** [comment]
It's hard to chose between 📦 and ⬆️

File: NuKeeper/Engine/RepositoryUpdater.cs [hunk 1]
```diff
@@ -98,7 +100,7 @@ private void GitCloneToTempDir()
                 Console.WriteLine("Commiting");
 
                 var commitMessage = CommitReport.MakeCommitMessage(updateSet);
-                _git.Commit(commitMessage);
+                _git.Commit($":{CommitEmoji}: {commitMessage}");
```
Comment (AnthonySteele) [inline NuKeeper/Engine/RepositoryUpdater.cs]: Should go in the `CommitReport.MakeCommitMessage` method?

Comment (slang25) [inline NuKeeper/Engine/RepositoryUpdater.cs]: The reason I didn't was cause you can't use emojis for PR titles, so I'm being lazy.
How about `MakeCommitMessage` returns a type with a method that prints it, and it takes an enum to distinguish between commit message and PR title?

Comment (AnthonySteele) [inline NuKeeper/Engine/RepositoryUpdater.cs]: Ah right, it's used in 2 places. How about 2 methods, `MakeCommitMessage` which forwards most to `MakePrTitle`.

Comment (slang25) [inline NuKeeper/Engine/RepositoryUpdater.cs]: Done

**Comment by slang25** [comment]
Ffs

File: NuKeeper/Engine/CommitReport.cs [hunk 2]
```diff
@@ -5,12 +5,18 @@
...
+    {
+        private const string CommitEmoji = "package";
+        public static string MakePullRequestTitle(PackageUpdateSet updates)
         {
             return $"Automatic update of {updates.PackageId} to {updates.NewVersion}";
         }
 
+        public static string MakeCommitMessage(PackageUpdateSet updates)
+        {
+            return $":{CommitEmoji}: {MakeCommitMessage(updates)}";
```
Comment (AnthonySteele) [inline NuKeeper/Engine/CommitReport.cs]: ^ recursion

**Comment by AnthonySteele** [comment]
Caught by the tests! Existing tests should be on `MakePullRequestTitle`, and maybe 1 or 2 tests on `MakeCommitMessage`

**Comment by slang25** [comment]
@AnthonySteele I got my act together

**Pull request merged**